src/nvim/buffer.c

```diff
@@ -2059,10 +2037,8 @@ void buflist_getfpos(void)
   }
 }
 
-/*
- * Find file in buffer list by name (it has to be for the current window).
- * Returns NULL if not found.
- */
+/// Find file in buffer list by name (it has to be for the current window).
+/// Returns NULL if not found.
```

Comment:
`@return buffer or NULL if not found` (and similar for functions below)

Reply:
I could've sworn I'd fixed all doxygen comments yesterday O_O . Thanks.